src/pages/dataset-visualization-view/dataset-visualization-view.test.js

```diff
@@ -3,6 +3,19 @@ import { shallow } from 'enzyme'
 import DatasetVisualizationView from './dataset-visualization-view'
 import ChartVisualization from '../../components/visualizations/chart/chart-visualization'
 import { Collapse } from 'react-collapse'
+import DatasetQuery from '../../components/dataset-query';
+import LoadingElement from '../../components/generic-elements/loading-element'
+
+const tableName = 'org1__table2'
+const expectedQuery = `SELECT * FROM ${tableName}\nLIMIT 20000`
+
+// Currently, shallow rendering is not compatible with React hooks.
+// We've utilized a strategy found here https://blog.carbonfive.com/2019/08/05/shallow-testing-hooks-with-enzyme/
```

Comment:
I appreciate these comments regarding the shallow rendering.

Reply:
Ah, starting to understand now.... is there any reason not to use `mount` rather than `shallow` as the 'short-term' fix for this issue?

Reply:
Other than "`mount` can turn into a rabbit hole of mocking" I don't think there is a good reason